docs/ref/models/fields.txt

```diff
@@ -2266,6 +2201,113 @@ accepted by :class:`ForeignKey`, plus one extra argument:
 See :doc:`One-to-one relationships </topics/db/examples/one_to_one>` for usage
 examples of ``OneToOneField``.
 
+.. _lazy-relationships:
+
+Lazy relationships
+------------------
+
+Lazy relationships allow you to reference models using their names as strings
+or to create recursive relationships. You can use strings as the first
+argument in any relationship field to reference models lazily. The following
+types of lazy references are available:
```

Comment:
In general, I tend to suggest to avoid this type of constructs (using "you"). I think the following is a bit more clear or professional:
```suggestion
Lazy relationships allow referencing models by their names (as strings) or
creating recursive relationships. Strings can be used as the first argument in
any relationship field to reference models lazily.

A lazy references can be either recursive, relative or absolute.
```
(We should definitely add links to each reference type.)

Reply:
So do I, but because Django docs use the second POV so much (sometimes informally), I am always unsure whether to use the professional tone (third POV), or else to preserve consistency and write in the same tone used in surrounding docs. Maybe we ought to include something in the style guides.

I applied the changes here though.